src/components/battleItemContainer.html

```diff
@@ -58,8 +68,21 @@
         </tbody>
       </table>
     </div>
-    <div class="card-header p-0" data-toggle="collapse" href="#fluteItemContainerBody" data-bind="visible: App.game.badgeCase.hasBadge(BadgeEnums.Heat)">
-      <span>Flutes</span>
+    <div data-bind="visible: App.game.badgeCase.hasBadge(BadgeEnums.Heat)">
+        <button type="button" class="btn btn-sm btn-primary"
+                style="position: absolute; left: 0px; width: auto; height: 41px"
+                onclick="ItemHandler.useAllFlutes(true)"
+                oncontextmenu="ItemHandler.useAllFlutes(false); return false"
+                title="Activate/Deactivate all Flutes"
+                data-toggle="popover"
+                data-html="true"
+                data-content="Left/right click to deactivate/activate all Flutes at once"
```

Comment:
I don't like that we use right-click as a part of the gameplay. People are used to right-click not doing anything in browser games, other than giving you a weird menu, and we have not used it so far, in this game.
Instead i would suggest it turned into "Deactivate all", when all unlocked flutes was activated

Reply:
Right click is already used to decrease the multiplier of battle items, so it seems logical to me to use this function. And with right click you can deactivate all the flutes even if they are not all activated.

Reply:
Oh it is? You learn something new each day!
But it's still not intuitive. What if we do a mix? Right click always deactivates. Left click deactivates if all of them are active. Else it will activate.